src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -36,7 +36,8 @@ def _query_account_key(cli_ctx, account_name):
     t_storage_account_keys = get_sdk(
         cli_ctx, ResourceType.MGMT_STORAGE, 'models.storage_account_keys#StorageAccountKeys')
 
-    scf.config.enable_http_logger = False
+    if hasattr(scf, 'config'):
+        scf.config.enable_http_logger = False
```

Comment:
Track2:
```
return scf.storage_accounts.list_keys(rg, account_name, logging_enable=False).key1

Reply:
Got it, it seems a CLI bug.

Reply:
I would consider that a bug indeed, but impact on CLI is limited since the process doesn't stay up very long.
In SDK usage, for a script that should run days or weeks uninterrupted, that would have killed the log system.